cmd/downloader.go

```diff
@@ -51,7 +51,7 @@ var downloadEVECmd = &cobra.Command{
 		if devModel == defaults.DefaultGCPModel {
 			format = "gcp"
 		}
-		if err := utils.DownloadEveLive(adamDist, eveImageFile, eveArch, eveHV, eveTag, eveUefiTag, format); err != nil {
+		if err := utils.DownloadEveLive(adamDist, eveImageFile, eveArch, eveHV, eveTag, eveUefiTag, format, eveImageSizeMB); err != nil {
```

Comment:
Can we find out it ourselves?

Reply:
We need to do it on step of setup. Cannot catch your idea, please elaborate.